### PR TITLE
Add jobs panel to session page

### PR DESCRIPTION
**Pull request by mgrauer**

### CODE REVIEW

**Comment by mgrauer** [comment]
This is failing because of a bitwise operator in JS.  Once https://github.com/girder/girder/pull/918 gets merged in Girder, Travis can be rerun and should pass.

File: web_external/js/views/body/JobsPanel.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    job: job,
+                    el: $('#g-dialog-container'),
+                    parentView: this
+                });
+                this.jobDetailsWidgetModalWrapper.render();
+            }, this);
+            this.jobListWidget.collection.fetch({}, true);
+        }, this);
+
+        girder.events.once('m:job.created', function () {
```
Comment (kotfic) [inline web_external/js/views/body/JobsPanel.js]: @mgrauer Is there a reason this is girder.events.once?  I've had to change it to girder.events.on  in order to get multiple jobs to show up without page refresh.

**Comment by mgrauer** [comment]
@kotfic no good reason.  fix pushed.

**Comment by kotfic** [comment]
LGTM

**Comment by aashish24** [comment]
:clap:

**Pull request merged**